cli/docker/docker.go

```diff
@@ -133,13 +135,20 @@ func (handler *Handler) GetImageFlavor(c config.JobConfiguration) string {
 	if c.Image.Version != "" {
 		tag = c.Image.Version
 	}
-	return fmt.Sprintf("%s:%s", c.Image.Base, tag)
+	defaultRegistry := "docker.io"
+	imageName := fmt.Sprintf("%s:%s", c.Image.Base, tag)
+        match, _ := regexp.MatchString(`.+\/.*\/.*`, imageName)
```

Comment:
Odd indentation, unless it's just github showing it that way 🤔

Reply:
vim 🤦